packages/vuetify/src/components/VNavigationDrawer/VNavigationDrawer.tsx

```diff
@@ -0,0 +1,142 @@
+// Styles
+import './VNavigationDrawer.sass'
+
+// Composables
+import { makeBorderProps, useBorder } from '@/composables/border'
+import { makeBorderRadiusProps, useBorderRadius } from '@/composables/border-radius'
+import { makeDimensionProps, useDimension } from '@/composables/dimensions'
+import { makeElevationProps, useElevation } from '@/composables/elevation'
+import { makeLayoutItemProps, useLayoutItem } from '@/composables/layout'
+import { makePositionProps, usePosition } from '@/composables/position'
+import { useProxiedModel } from '@/composables/proxiedModel'
+import { useTheme } from '@/composables/theme'
+
+// Utilities
+import { computed, defineComponent, onBeforeMount, ref, toRef } from 'vue'
+import { convertToUnit } from '@/util/helpers'
+import { makeTagProps } from '@/composables/tag'
+import makeProps from '@/util/makeProps'
+
+export default defineComponent({
+  name: 'VNavigationDrawer',
+
+  props: makeProps({
+    ...makeBorderProps(),
+    ...makeBorderRadiusProps(),
+    ...makeDimensionProps({ width: 256 }),
+    ...makeElevationProps(),
+    ...makeLayoutItemProps(),
+    ...makePositionProps(),
+    ...makeTagProps({ tag: 'nav' }),
+    expandOnHover: Boolean,
+    mobile: Boolean,
+    modelValue: Boolean,
+    permanent: Boolean,
```

Comment:
I'm still not convinced of this prop, but 🤷. I feel like the proxy composable was made specifically for this exact scenario, where a user wants to absolutely control some state of a component without the component itself doing something that makes the internal value change.

Reply:
There is a certain level of simplicity that comes from a boolean prop. This would be a good example usage of prop aliases. For now I'd like to keep it and revisit if/when necessary.

Reply:
The alternative would also just be one prop `:model-value="true"`